app/controllers/public_health_controller.rb

```diff
@@ -278,6 +290,100 @@ def linelist_specific_fields(workflow, tab)
     %i[jurisdiction assigned_user end_of_monitoring risk_level monitoring_plan latest_report report_eligibility]
   end
 
+  def advanced_filter(patients, filters)
+    filters.each do |filter|
+      case filter[:filterOption]['name']
+      when 'sent-today'
+        patients = patients.where("last_assessment_reminder_sent #{filter[:value] ? '>' : '<'} ?", DateTime.now.beginning_of_day)
+      when 'responded-today'
+        patients = patients.where("latest_assessment_at #{filter[:value] ? '>' : '<'} ?", DateTime.now.beginning_of_day)
```

Comment:
Can we assume that when they say "today" they are referring to today in their timezone rather than in UTC? If so this would need to be changed.